### PR TITLE
SL-1155: Allow to refresh the access token.

### PR DESCRIPTION
**Pull request by migueltaoliveira**

### CODE REVIEW

File: lib/workable/client.rb [hunk 1]
```diff
@@ -257,6 +257,19 @@ def event(id)
       @transform_to.apply(:event, get_request("events/#{id}"))
     end
 
+    def refresh_access_token(client_id, client_secret, refresh_token)
+      uri = URI.parse('https://www.workable.com/oauth/token')
```
Comment (dmbento) [inline lib/workable/client.rb]: Can you move this to an EnvVar or a constant?

Comment (dmbento) [reply lib/workable/client.rb]: I think we can use `post_request` method available.

Comment (migueltaoliveira) [reply lib/workable/client.rb]: The `post_request` is expecting the default headers as well as the api_url.
Not sure if we want to refactor that method, which will diverge from the forked repository. I will move the URL to a constant. Thanks for the review!

Comment (dmbento) [reply lib/workable/client.rb]: https://github.com/ScreenLoop/workable-rb/blob/e74b57ac7c4b43ab5614ef3a97397d0fdfda8d70/lib/workable/client.rb#L292

`post_request` accepts a block and you can change the headers there

Comment (migueltaoliveira) [reply lib/workable/client.rb]: Not sure if I get your point.
In that case, the do_request is called, the block I can change is this:
`yield(request) if block_given?`

Comment (dmbento) [reply lib/workable/client.rb]: No need to change after discussing this IRL

**Review by dmbento — APPROVED** [review]

**Pull request merged**